src/Utilities/TypeTraits.hpp

```diff
@@ -906,6 +906,39 @@ using is_callable_t = typename is_callable<T, Args...>::type;
   using is_##METHOD_NAME##_callable_t =                                        \
       is_##METHOD_NAME##_callable_r_t<AnyReturnType##METHOD_NAME, T, Args...>;
 
+/*!
+ * \ingroup TypeTraitsGroup
+ * \brief Generate a type trait to check if a class has a `static constexpr`
+ * variable, optionally also checking its type.
+ *
+ * \example
+ * \snippet Utilities/Test_TypeTraits.cpp CREATE_HAS_EXAMPLE
+ *
+ * \see `CREATE_IS_CALLABLE`
+ */
+#define CREATE_HAS(CONSTEXPR_NAME)                                           \
```

Comment:
`HAS_MEMBER_VARIABLE` and below

Reply:
Is "member variable" the correct term for a `static constexpr` variable?

Reply:
to be clear, what are you asking to rename specifically?

- `CREATE_HAS ` -> `CREATE_HAS_MEMBER_VARIABLE `
- or `CONSTEXPR_NAME ` -> ?

Reply:
Yes, they are member variables. You could specify `CREATE_HAS_STATIC_MEMBER_VARIABLE` since that's what it really checks for.